datafusion/src/physical_plan/sort_preserving_merge.rs

```diff
@@ -963,6 +963,10 @@ mod tests {
                 expr: col("c7", &schema).unwrap(),
                 options: SortOptions::default(),
             },
+            PhysicalSortExpr {
+                expr: col("c12", &schema).unwrap(),
```

Comment:
added to make the sort output deterministic

Without this change, there is one row that comes out in a slightly different order (same values of `c2` and `c7`)

![Screen Shot 2021-10-18 at 12 55 18 PM](https://user-images.githubusercontent.com/490673/137777649-b26f63ab-c84d-40b7-8202-1061fe091cbb.png)